__tests__/app.test.js

```diff
@@ -311,4 +322,21 @@ describe('DELETE /api/comments/:comment_id', () => {
                 expect(response.body.msg).toBe('bad request')
             })
     })
-});
\ No newline at end of file
+});
+
+describe('GET /api/users', () => {
+    it('should respond with an array of all users', () => {
+        return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then((response) => {
+                const {users} = response.body
+                    users.forEach((user)=>{
+                        expect(user).toHaveProperty('username')
+                        expect(user).toHaveProperty('name')
+                        expect(user).toHaveProperty('avatar_url')
+                    })
```

Comment:
You should have an expect for a length check on the array before asserting inside this forEach